ui/src/app/components/gke-app-configuration-panel.spec.tsx

```diff
@@ -66,7 +84,20 @@ describe(GKEAppConfigurationPanel.name, () => {
     expect(wrapper.childAt(0).is(Spinner)).toBeFalsy();
   });
 
-  it('should show an error if the API call fails', async () => {
+  it('should show a loading spinner while waiting for the list disks API call to return', async () => {
```

Comment:
It's difficult to use `each` here since `appsApi` is a function and we need to initialize the APIs before calling it. However, I was able to move most of the code from each into a helper function which I think fine, too.